cfg/builder/builder_entry.cc

```diff
@@ -36,13 +36,82 @@ unique_ptr<CFG> CFGBuilder::buildFor(core::Context ctx, ast::MethodDef &md) {
                                          selfClaz.data(ctx)->enclosingClass(ctx).data(ctx)->selfType(ctx),
                                          core::Names::cast()));
 
+        BasicBlock *presentCont = entry;
+        BasicBlock *defaultCont = nullptr;
+
+        auto &argInfos = md.symbol.data(ctx)->arguments();
+        bool isAbstract = md.symbol.data(ctx)->isAbstract();
+        bool seenKeyword = false;
         int i = -1;
         for (auto &argExpr : md.args) {
             i++;
             auto *a = ast::MK::arg2Local(argExpr);
-            synthesizeExpr(entry, res->enterLocal(a->localVariable), a->loc, make_unique<LoadArg>(md.symbol, i));
+            auto local = res->enterLocal(a->localVariable);
+            auto &info = argInfos[i];
```

Comment:
When i was looking at `info.flags.foo` below, I didn't know that this was an `ArgInfo` struct. Would you mind renaming this to `argInfo`?
```suggestion
            auto &argInfo = argInfos[i];
```